sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/BinaryDataJsonSerializer.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.implementation;
+
+import com.azure.core.util.BinaryData;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+import java.util.Base64;
+
+public class BinaryDataJsonSerializer extends JsonSerializer<BinaryData> {
```

Comment:
This serializer and deserializer above are generic. Can this be moved to azure-core? 
cc: @alzimmermsft

Reply:
My general feeling to seeing this is wishing that we wouldn't introduce new use cases requiring databind - it is just another thing we need to unwind in the future. Should we not have a policy that suggests when a new databind use is needing to be introduced that instead we begin efforts to unwind it instead?